bundles/org.openhab.binding.tado/src/main/java/org/openhab/binding/tado/internal/handler/TadoZoneHandler.java

```diff
@@ -282,8 +282,8 @@ private void scheduleHvacChange() {
             } catch (IOException e) {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
             } catch (ApiException e) {
-                logger.error("Could not apply HVAC change on home {} and zone {}: " + e.getMessage(), e, getHomeId(),
-                        getZoneId());
+                logger.error("Could not apply HVAC change on home {} and zone {}: {}", getHomeId(), getZoneId(),
```

Comment:
Maybe also change log level to warn? I don't know if the stack trace might still be relevant here, but in that case add `e`.